index.js

```diff
@@ -3,30 +3,84 @@
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @returns {boolean} Returns `true` if all values in the array are numbers. Otherwise, return `false`.
  */
-function isValid(rolls) {}
+function isValid(rolls) {
+
+let flag = true
+if(rolls.length === 0) {
+  return true
+}
+  for (let i = 0; i < rolls.length; i++) {
+    if(typeof rolls[i] === 'number') {
+      return flag
+    } 
+    flag = false
+    return flag
+  }
+  
+}
+
+//return rolls.every(element => typeof element === 'number')
 
 /**
  * Finds a value in an array. If that value is in the array, returns it. Otherwise, returns `null`.
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @param {number} value - A specific value to find.
  * @returns {*} - The found value or `null`.
  */
-function findValue(rolls, value) {}
+function findValue(rolls, value) {
+  
+ if(rolls.length ===0) {
+  return null
+ } if(rolls.includes(value)) {
+  return value
+}   else {
+  return null
+}
+    }
+    
```

Comment:
Be careful with your formatting. The closing bracket to the function should line up with the function keyword.